docs/sources/write/links/_index.md

```diff
@@ -35,7 +35,27 @@ Although these other types of links still function, replace them with one of the
 
 ## Link from source content that's reused as multiple pages
 
-Use `ref` URIs.
+Use a full URL and ensure the mount target has `mount_links: true` in its front matter.
+The `mount_links` front matter tells Hugo to link to mounted content in the current project if it exists.
+
+During link rendering, Hugo checks if the link destination is a mounted source.
+If it's mounted, then Hugo checks if the current page is in the same project as the mount target.
+If it is in the same project, then Hugo replaces the link destination with the mount target.
+If it isn't mounted, and the mount target isn't in the same project, then Hugo uses the link as normal.
+
+### Mount links example
+
+The Grafana Labs website configuration mounts the open source Grafana Alerting documentation into the Grafana Cloud documentation.
+
+From the Grafana Alerting introduction page, the link `[alert rule](/docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/alert-rules/)` has different destinations in the open source and cloud versions of the page.
```

Comment:
The directions above say to use a full URL, but that doesn't seem to be what's in this example.

Reply:
Good call out!

Reply:
Good point, I'll just refer to URLs.

I think we've coined the term full URL to refer to a URL with scheme (like `https://`), hostname (like `grafana.com`), and URL pathinfo (like `/docs/grafana/`). In our internal links, we only ever need the pathinfo.

Reply:
I think that makes sense. I've made a suggestion above just to give people some sense of the format without having to read the example.